src/openrct2/world/Footpath.cpp

```diff
@@ -1891,7 +1891,8 @@ void footpath_update_path_wide_flags(const CoordsXY& footpathPos)
         if (tileElement->AsPath()->GetEdges() & EDGE_NW)
         {
             pathConnections |= FOOTPATH_CONNECTION_NW;
-            if (std::get<3>(pathList) != nullptr && std::get<3>(pathList)->AsPath()->IsWide())
+            const auto& path_element = std::get<3>(pathList);
```

Comment:
```suggestion
            const auto* pathElement = std::get<3>(pathList);
```
Use auto* when its a pointer and auto& when its a reference and just auto when you intent to do a copy. Also our coding style uses ```camelCase``` for variables.

Reply:
Got it, thank you for that feedback